Tests/Functional/config/framework.yml

```diff
@@ -2,7 +2,7 @@ framework:
     secret: test
     test: true
     router:
-        resource: "%kernel.root_dir%/config/routing.yml"
+        resource: "%kernel.project_dir%/config/routing.yml"
```

Comment:
`kernel.project_dir` is not direct replacement for `kernel.root_dir`. Root dir usually pointed at `app/` directory so it has to be used with `../`. Anyway, it won't work here because `routing.yml` file doesn't exist. It has to be rewritten for new directory and configuration structure. Can you check if this is feasible for you? If not you could add a comment that the tests are broken here.

Reply:
I tried to fix it but there are a lot of other problems, at the moment tests are broken

Reply:
Actually we don't need this setting at all. This configuration is only for  functional test  of commands. Dropped in d33d1ed